terraform/environments/mlra/cloudwatch.tf

```diff
@@ -0,0 +1,9 @@
+odule "cwalarm_1" {
+  source = "./modules/cloudwatch"
+
+  pClusterName          = " "
+  pAutoscalingGroupName = " "
+  pLoadBalancerName     = " "
+  pTargetGroupName      = aws_lb_target_group.alb_target_group.name
+  appnameenv            = "${local.application_name}-${local.environment}"
+}
```

Comment:
Once it is verified the Terraform plan is good and do what you expected, remove this file from this commit please since we still have dependencies on resources not yet created from other modules (thus result in failed deployment).
Also there is a small typo on line 1.

Reply:
moved to main and comented out for now